styles/mixins/button.js

```diff
@@ -0,0 +1,10 @@
+module.exports = function () {
+  return function (mixin, bgColor, color) {
+    return {
+      'background-color': bgColor,
+      color,
+      display: 'inline-block',
+      'line-height': 1
```

Comment:
I know it's gross, but I'd only do that to an attributed selected input element.

```
input[type="button"],
Input[type="submit"]
```